code/datums/brain_damage/special_ed.dm

```diff
@@ -90,4 +90,95 @@
 		new /obj/effect/overlay/temp/bluespace_fissure(get_turf(linked_to))
 		user.forceMove(get_turf(linked_to))
 		user.visible_message("<span class='warning'>[user] [slip_in_message].</span>", ignored_mob = user)
-		user.visible_message("<span class='warning'>[user] [slip_out_message].</span>", "<span class='notice'>...and find your way to the other side.</span>")
\ No newline at end of file
+		user.visible_message("<span class='warning'>[user] [slip_out_message].</span>", "<span class='notice'>...and find your way to the other side.</span>")
+
+/datum/brain_trauma/special/love
+	name = "Hyper-dependency"
+	desc = "Patient feels lovesick and is emotionally dependent to a specific person."
+	scan_desc = "severe dependency"
+	gain_text = ""
+	lose_text = "<span class='notice'>You feel love leave your heart.</span>"
+	var/stress = 0
+	var/datum/weakref/beloved = null
+	cure_type = CURE_HYPNOSIS
+
+/datum/brain_trauma/special/love/on_gain()
+	..()
+	for(var/mob/living/L in view(7,owner))
```

Comment:
How does in view work? Is the list random or is it based on something? Might want to shuffle in case view() sorts by something like memory ID, Name, or closest on the x-axis.

Reply:
wat.

It's probably deterministic, with the order depending specifically on what algorithm is being used, but it's most likely repeatable.

_But._ With that said. I don't see how this causes any issue? `on_gain` is called once per mob gaining the trauma. Just about the only time this would cause issues is when two people right next to each other gain the trauma at the exact same moment. Which means it's kind of negligible. ~And it might be hilarious RP.~